src/DependencyInjection/Configuration.php

```diff
@@ -173,6 +173,29 @@ public function getConfigTreeBuilder()
                             ->info('Background used in mosaic view')
                         ->end()
+                        // NEXT_MAJOR : remove this option
+                        ->booleanNode('legacy_twig_text_extension')
+                            ->info('Use text filters from "twig/extensions" instead of those provided by "twig/string-extra".')
+                            ->defaultValue(static function (): bool {
+                                @trigger_error(
```

Comment:
Maybe [we could use `setDeprecated` method](https://symfony.com/doc/current/components/config/definition.html#deprecating-the-option) here.

Reply:
I think that's for deprecating an entire node. We want to deprecate not setting it or setting it to true here. Using it with false should be fine.

Reply:
The option is new, what is deprecated is using other value than `false`.
If we deprecate the option too, the user will have no way to fix both deprecations (the usage of the old `truncate` behavior and the presence of this option).